src/components/tooltip/Tooltip.tsx

```diff
@@ -231,6 +231,9 @@ class Tooltip extends React.Component<TooltipProps, State> {
 
         if (showTooltip) {
             componentProps['aria-describedby'] = this.tooltipID;
+            if (theme === TooltipTheme.ERROR) {
+                componentProps['aria-errormessage'] = this.tooltipID;
```

Comment:
Hmm, so wouldn't this still need to be on the input? The input is the source element that is in the error state, not the tooltip itself, and with this change now this tooltipID contains the target of the `aria-errormessage`?

Reply:
Yes. `aria-errormessage` along with `aria-describedby` is on the input. It references the tooltip that contains the actual error message text `Field Required.`. 

Originally, it was set to the `errorMessageID` from `TextInput.js` which had the same error message text. Now that this is removed from `TextInput.js`, I changed it to set it from the `Tooltip.tsx` where I have the tooltip ID.

Reply:
Ah yep I see now. Misunderstood where `componentProps` were getting used.